regression_detector.py

```diff
@@ -128,6 +141,8 @@ def process_regressions_into_gh_issue(regressions_dict, owner: str, output_path:
         "test_details": troubled_tests,
         "control_only_tests": control_only_tests,
         "treatment_only_tests": treatment_only_tests,
+        "runtime_regressions_msg": runtime_regressions_msg,
+        "github_run_id": os.environ.get("GITHUB_RUN_ID", ""),
```

Comment:
If `GITHUB_RUN_ID` is not set, we will print an invalid URL, maybe print something else when it is not set?

Reply:
Okay!